daemon/execdriver/native/seccomp_unsupported.go

```diff
@@ -0,0 +1,10 @@
+// +build linux,!seccomp
+
+package native
+
+import "github.com/opencontainers/runc/libcontainer/configs"
+
+var (
+	seccompSupported      = false
+	defaultSeccompProfile = &configs.Seccomp{}
```

Comment:
ya it doesnt like nil, that was my first choice tho then got the type error
:(

On Mon, Dec 28, 2015 at 8:33 PM, Michael Crosby <notifications@github.com>
wrote:

> In daemon/execdriver/native/seccomp_unsupported.go
> <https://github.com/docker/docker/pull/18947#discussion_r48521165>:
>
> > @@ -0,0 +1,10 @@
> > +// +build linux,!seccomp
> > +
> > +package native
> > +
> > +import "github.com/opencontainers/runc/libcontainer/configs"
> > +
> > +var (
> > +	seccompSupported      = false
> > +	defaultSeccompProfile = &configs.Seccomp{}
>
> could this just be nil and then you could probably remove the need for
> the boolean. Anyways, nice job on this ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18947/files#r48521165>.
>